dll/win32/shell32/lang/en-US.rc

```diff
@@ -191,7 +191,7 @@ BEGIN
     LTEXT "Shortcut &key:", IDC_SHORTCUT_KEY, 8, 117, 68, 10
     CONTROL "", IDC_SHORTCUT_KEY_HOTKEY, "msctls_hotkey32", WS_VISIBLE | WS_CHILD | WS_BORDER | WS_TABSTOP, 79, 115, 150, 14
     LTEXT "&Run:", IDC_SHORTCUT_RUN, 8, 136, 68, 10
-    COMBOBOX IDC_SHORTCUT_RUN_COMBO, 79, 134, 150, 14, CBS_DROPDOWNLIST | WS_CHILD | WS_VISIBLE | WS_VSCROLL | WS_TABSTOP
+    COMBOBOX IDC_SHORTCUT_RUN_COMBO, 79, 134, 150, 54, CBS_DROPDOWNLIST | WS_CHILD | WS_VISIBLE | WS_VSCROLL | WS_TABSTOP
```

Comment:
Why is this necessary? Is this because otherwise we have to scroll in the list to "see" the three choices?

Reply:
It is the classic combo bug. The height of the control specifies the height of the drop list, the control itself forces the size based on the font no matter what you do (And ComboBoxEx tries and fails to work around this). The bug is only visible with ComCtl v5 on Windows IIRC.

Reply:
Ah, so it is also present on "old" Windows, nice to know that. :+1: